src/app/app.routes.ts

```diff
@@ -42,6 +42,13 @@ export const routes: Routes = [
     loadComponent: () =>
       import('./features/post/post.component').then(m => m.PostComponent),
   },
+  {
+    path: 'authors',
```

Comment:
Can we use `people`? This way we have both `people` and `people/:author` – see above

Reply:
Done